packages/runtime/jest.config.ts

```diff
@@ -28,7 +28,6 @@ const config: Config = {
       },
     ],
   },
-  snapshotSerializers: ['@emotion/jest/serializer'],
```

Comment:
Moved to a programmatic Emotion snapshot serializer setup, see [below](https://github.com/makeswift/makeswift/pull/929/files#diff-6726608b780ee23cc29b0be0fba5278a6618efb98c8a08e46352e97b74c905bbR10).